MSAL/src/cache/MSALBaseTokenCacheItem.h

```diff
@@ -36,7 +36,7 @@
 @interface MSALBaseTokenCacheItem : MSALJsonObject
 
 @property NSString *clientId;
-@property (readonly) MSALClientInfo *clientInfo;
+@property (readonly) MSIDClientInfo *clientInfo;
```

Comment:
do we still need MSALBaseTokenCacheItem?

Reply:
removed